code/style.css

```diff
@@ -1,7 +1,431 @@
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  position: absolute;
+}
+
+#hero {
+  width: 100vw;
+  height: 100vh;
+  overflow-x: hidden;
+  margin: 0;
+}
+
+#hero video {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 10;
+}
+
+#hero .headerText, .navbar {
+  position: relative;
+  z-index: 20;
+}
+
+.flexParentHeader {
+  display: flex;
+  flex-direction: column;
+}
+
+.headerText {
+  display: flex;
+  width: 100%;
+  margin: auto;
+  margin-top: 30px;
+  justify-content: center;
+  font-family: "permanent marker";
+  font-size: 34px;
+  color: #FFF7AE;
+  text-shadow: 2px 2px #9D98FA;
+}
+
+/* NAVBAR */
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  width: 100vw;
+  margin: auto;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 30px;
+}
+
+ul {
+  width: 100vw;
+  display: flex;
+  list-style-type: none;
+  justify-content: center;
+  padding-inline-start: 0;
```

Comment:
Interesting trick! I understood its functionality once I deactivated it in the inspector.

Reply:
I was losing my mind when trying to center elements and finally found that there was a "padding-inline-start" value by default (!!)